pkg/certificate/mrc_reconciler.go

```diff
@@ -130,7 +131,12 @@ func shouldUpdateState(mrc *v1alpha2.MeshRootCertificate) bool {
 // update the MRCComponentStatus
 func (m *Manager) updateMRCComponentStatus(mrc *v1alpha2.MeshRootCertificate) error {
 	// TODO(5046): determine the status to set the owned components to.
-	var status v1alpha2.MeshRootCertificateComponentStatus
+	// default component status is Unknown
+	var status v1alpha2.MeshRootCertificateComponentStatus = constants.MRCComponentStatusUnknown
```

Comment:
nit: I think this could be simplified to the following:
```suggestion
	status := constants.MRCComponentStatusUnknown
```

Reply:
the `constants.MRCComponentStatusUnknown` is a string so I set it the above way so that it can be type `v1alpha2.MeshRootCertificateComponentStatus`. I think instead I'll change the constants to be their specific type